cli/admin/src/main/java/io/pravega/cli/admin/utils/CLIControllerConfig.java

```diff
@@ -97,6 +104,7 @@ private CLIControllerConfig(TypedProperties properties) throws ConfigurationExce
         this.userName = properties.get(CONTROLLER_USER_NAME);
         this.password = properties.get(CONTROLLER_PASSWORD);
         this.truststore = properties.get(TRUSTSTORE_JKS);
+        this.accessTokenTtlInSeconds = properties.getInt(TRUSTSTORE_ACCESS_TOKEN_TTL_SECONDS);
```

Comment:
The best practice (at least in our codebase), is to store all time durations as a `Duration` object. Check out other config objects in our codebase (like `ContainerConfig`, `WriterConfig`, etc) to see how to use those. 

I even believe that `TypedProperties` has a `getDuration` method to help you out.